### PR TITLE
React todo app 

### PR DESCRIPTION
**Pull request by vvoo21**
From the React tutorial, these are the completed changes ⚙️🛠️:
- React application initialized
- Create the component files
- Use class-based components and functional components to describe the user interface
- Work with event handler
- Add CSS in the React app with the inline CSS, CSS style sheet, and CSS modules in this part.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -10,34 +10,11 @@
...
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React App</title>
```
Comment (lmunozm1702) [inline public/index.html:27]: Nice work cleaning the comment code!

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,65 @@
-.App {
```
Comment (lmunozm1702) [inline src/App.css:1]: Great job cleaning the styles, you let only the relevant components!

**Review by lmunozm1702 — APPROVED** [review]
HI @vvoo21 great work with the tutorial, your code is very clear and easy to read! 

:high_brightness: The best:
- Linters passed.
- Clean code.
- Great comments on commits and PR.
- All works as expected.

:confetti_ball:  Then, now you can MERGE the code. :confetti_ball:

**Comment by vvoo21** [comment]
@lmunozm1702 Thanks for your code review 👍

**Pull request merged**